module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -1674,6 +1677,77 @@ public function getCancelHoldDetails($hold, $patron = [])
             ? $hold['reqnum'] : '';
     }
 
+    /**
+     * Get Update Hold Details
+     *
+     * Get required data for updating a hold. This value is relayed to the
```

Comment:
Should the comment make more clear that an empty string means the hold is not eligible for modification? Or am I overlooking/misunderstanding something?

Reply:
Now the method doesn't exist anymore. Need to document the field returned in getMyHolds.